app/helpers/curate_helper.rb

```diff
@@ -182,11 +177,24 @@ def sorted_college_list_for_generic_works
       list[k]["label"]
     end.sort << "Other"
   end
-
+    def sorted_college_list_for_degrees
```

Comment:
Sean, I think i did this initially, thought I would need it, but I dont believe I do, do you agree I can just delete this- as well as line below? Agree?